### DISCUSSION TITLE
BIRD 1.6.8 .tgz missing

### DISCUSSION THREAD

**Discussion by Dukat-Gul** [Freetz-NG/freetz-ng#1031] [Allgemein] [answered]
Hi, I have been successful building BIRD [ver 2.15-1] for by 7530 and 6660, but not for my 7490.
Ill create a separate discussion for that.

I wanted to try the older Bird [ver 1.6.8] but it seems to not be available to download. [Even in the  [freetz-dl-mirror].](https://github.com/Freetz-NG/dl-mirror)

Same for all the other attempted sites.

Download failed - "https://bird.network.cz/download/bird-1.6.8.tar.gz"  ->  error code 8
```
--2024-08-16 17:54:00--  ftp://bird.network.cz/pub/bird/bird-1.6.8.tar.gz
           => ‘dl/bird-1.6.8.tar.gz’
Resolving bird.network.cz... 81.91.84.116, 2001:1568:b:145::1, 2001:1568:b::145
Connecting to bird.network.cz|81.91.84.116|:21... connected.
Logging in as anonymous ... Logged in!
==> SYST ... done.    ==> PWD ... done.
==> TYPE I ... done.  ==> CWD (1) /pub/bird ... done.
==> SIZE bird-1.6.8.tar.gz ... done.
==> PASV ... done.    ==> RETR bird-1.6.8.tar.gz ...
No such file ‘bird-1.6.8.tar.gz’.
```
Download failed - "ftp://bird.network.cz/pub/bird/bird-1.6.8.tar.gz"  ->  error code 8
```
Resolving freetz-ng.root.sx... 212.132.112.203, 2a01:239:2d3:2400::babe
Connecting to freetz-ng.root.sx|212.132.112.203|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-08-16 17:54:07 ERROR 404: Not Found.

```
Download failed - "https://freetz-ng.root.sx/bird-1.6.8.tar.gz"  ->  error code 8
```
Resolving freetz.magenbrot.net... 31.172.113.113
Connecting to freetz.magenbrot.net|31.172.113.113|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-08-16 17:54:10 ERROR 404: Not Found.
```
Download failed - "https://freetz.magenbrot.net/bird-1.6.8.tar.gz"  ->  error code 8
```
Resolving freetz.wirsind.info... 188.165.115.52
Connecting to freetz.wirsind.info|188.165.115.52|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-08-16 17:54:13 ERROR 404: Not Found.
```
Download failed - "https://freetz.wirsind.info/bird-1.6.8.tar.gz"  ->  error code 8
```
Resolving raw.githubusercontent.com... 185.199.111.133, 185.199.108.133, 185.199.110.133, ...
Connecting to raw.githubusercontent.com|185.199.111.133|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-08-16 17:54:14 ERROR 404: Not Found.
```
Download failed - "https://raw.githubusercontent.com/Freetz-NG/dl-mirror/master/bird-1.6.8.tar.gz"  ->  error code 8

As a FYI - 
I think its just an incorrect path for the first site attempt.
[https://bird.network.cz/download/1.6/bird-1.6.8.tar.gz](https://bird.network.cz/download/1.6/bird-1.6.8.tar.gz )
exists. I downloaded and put it manually into my ~/.freetz-dl

**Comment by fda77** [comment, marked as answer]
thx, fixed:
* 08e61a9af bird: fix download url
